server/.gitignore

```diff
@@ -0,0 +1 @@
+.env.staging
```

Comment:
Creaste otro .gitignore aca, no te parece mejor ponerlo en el de root, poniendole `/server/.env.staging`

Reply:
pifie, tire el comando add to git ignore desde otra carpeta. Lo arreglo

Reply:
Vpe